guides/common/modules/ref_supported-operating-systems.adoc

```diff
@@ -1,53 +1,42 @@
 [id="supported-operating-systems_{context}"]
 = Supported Operating Systems
 
-ifeval::["{build}" == "satellite"]
+ifdef::satellite[]
 You can install the operating system from a disc, local ISO image, kickstart, or any other method that Red{nbsp}Hat supports.
 Red{nbsp}Hat {ProductName} is supported only on the latest versions of {RHEL} 7 Server that is available at the time when {ProductName} {ProductVersion} is installed.
 Previous versions of Red{nbsp}Hat Enterprise Linux including EUS or z-stream are not supported.
 endif::[]
 
```

Comment:
The satellite guide does not build after this point, so something has gone awry :)